### PR TITLE
Adding completed identity topic.

### PR DESCRIPTION
**Pull request by Erikre**

### CODE REVIEW

**Comment by Erikre** [comment]
@Rick-Anderson and @danroth27 - Please review. Thanks

**Comment by Rick-Anderson** [comment]
@rustd please review or assign for review ASAP - Identity

**Comment by rustd** [comment]
:shipit:

**Comment by Rick-Anderson** [comment]
@rachelappel  this is ready for the merge dance.

File: aspnet/security/introduction-to-aspnet-identity.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
+Introduction to ASP.NET Identity
+================================
+
+By `Pranav Rastogi`_, `Rick Anderson`_, Tom Dykstra, Jon Galloway, and `Erik Reitan`_
+
+ASP.NET Identity is a membership system which allows you to add login functionality to your application. Users can create an account and login with a user name and password or they can use an external login providers such as Facebook, Google, Microsoft Account, Twitter and more.
```
Comment (MikeWasson) [inline aspnet/security/introduction-to-aspnet-identity.rst]: an external login providers > delete "an"

**Comment by rachelappel** [comment]
Merged 2233e4dfca6ea7116fb9c08009b9d1ac7c3acd05

**Pull request closed without merging**